hydra-node/test/Hydra/Model.hs

```diff
@@ -711,7 +711,7 @@ performDecommit party tx = do
   party `sendsInput` Input.Decommit realTx
 
   lift $ do
-    waitUntilMatch [thisNode] $ \case
+    waitUntilMatch (Map.elems nodes) $ \case
```

Comment:
`toList :: Foldable f => f a -> [a]` is usually what I reach for (instead of container specific functions)